Automattic/wp-calypso
# Remove fullstory integration in Calypso

Context p58i-dUF-p2

## Proposed Changes

* Removes all tracking and scripts related to [FullStory](https://www.fullstory.com/) integration.

## Testing Instructions

* Smoke test everything works as expected across Calypso, /start, /setup/free, checkout form, etc.

## Pre-merge Checklist
